journal/journal.go

```diff
@@ -50,11 +50,11 @@ const (
 	PriDebug
 )
 
-var conn net.Conn
+var conn net.PacketConn
 
 func init() {
 	var err error
-	conn, err = net.Dial("unixgram", "/run/systemd/journal/socket")
+	conn, err = net.ListenPacket("unixgram", "")
```

Comment:
Any specific reasons to use `ListenPacket` in here? I have some concerns about it:
 * there is a more specific `ListenUnixgram` that perhaps could be used for clarity
 * I would expect a DialFoo function in here, as this is the client side of the communication
 * I tried to trace down what a `""` address means for a listening UNIX socket, but I'm still not sure about all the implications
 * I'm not sure what happens when multiple processes try to bind to this empty address, and what happens if this is imported multiple times in the same binary.